login.css

```diff
@@ -0,0 +1,42 @@
+.dropbtn {
+    color: black;
+}
+.dropdown {
+    position: absolute;
+    right: 2.5rem;
+    margin-bottom: 3.2rem;
+    font-size: 1.3em;  
+    color: var(--white); 
+}
+.dropdown-content {
+    position: absolute;
+    background-color: var(--blueviolet);
+    border-radius: 0.2rem;
+    right: 0.03rem;
+    top: 3.2rem;
+    padding: 0.5rem;
+    padding-left: 0.5rem;
+    display: none;
+    text-align: center;
+}
+.dropdown-content .a{
+    color: black;
```

Comment:
You should use variable for black color also as it would be more consistent.

Reply:
i forget to use variable in hurry, changing it.